src/app/components/chat/room-details-modal/form/form.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Emit } from 'vue-property-decorator';
+import { BaseForm } from '../../../../../_common/form-vue/form.service';
+import { ChatRoom } from '../../room';
+
+export type FormModel = {
```

Comment:
If we're exporting this, I think we should rename it to relate to the scope it represents so it doesn't get confusing when importing.